app/stats/status.rb

```diff
@@ -1,23 +1,79 @@
-require_relative "./level.rb"
-require_relative "./force.rb"
-require_relative "./defense.rb"
-require_relative "./hit_point.rb"
-require_relative "./magic_point.rb"
+require_relative "stats.rb"
+require_relative "hit_point.rb"
+require_relative "magic_point.rb"
+require_relative "force.rb"
+require_relative "defense.rb"
+require_relative "dexterity.rb"
+require_relative "luck.rb"
+
 
 module Stats
 
     class Status
-        attr_accessor :hit_point, :magic_point, :force, :defense
-        def initialize(hit_point=Stats::DEFAULT_HIT_POINT,
-                       magic_point=Stats::DEFAULT_MAGIC_POINT,
-                       force=Stats::DEFAULT_FORCE,
-                       defense=Stats::DEFAULT_DEFENSE)
+        attr_accessor :hit_point, :magic_point, 
+                      :force, :defense, :dexterity, :luck
+
+        def initialize(hit_point: Stats::DEFAULT_HIT_POINT,
+                       magic_point: Stats::DEFAULT_MAGIC_POINT,
+                       force: Stats::DEFAULT_FORCE,
+                       defense: Stats::DEFAULT_DEFENSE,
+                       dexterity: Stats::DEFAULT_DEXTERITY,
+                       luck: Stats::DEFAULT_LUCK)
             @hit_point = hit_point
             @magic_point = magic_point
             @force = force
             @defense = defense
+            @dexterity = dexterity
+            @luck = luck
+        end
+
+
```

Comment:
You should probably change the method name but this example shows a more idiomatic syntax that uses `each` loops rather than `for`

```ruby
    def to_s
      stat_values ||= []
      instance_variables.each { |name| stat_values << instance_variable_get(name) }

      stat_values
    end
```